src/nextjournal/clerk/sci_viewer.cljs

```diff
@@ -481,9 +481,8 @@
                                       :title (if (or (string? k) (keyword? k)) (name k) (str k))}
                                      [:div.inline-flex
                                       ;; Truncate to available col width without growing the table
-                                      [:div.table.table-fixed.w-full.flex-auto
-                                       {:style {:margin-left -12}}
```

Comment:
Hmmm, looking at examples in `notebooks/viewers/table.clj` this seems to align certain columns correctly. Most of the tables look weird without this. Eg.:
![image](https://user-images.githubusercontent.com/772738/161372454-a2b52499-6436-496f-a51c-541ae8820d4d.png)
But then my test looks strange with it: 
![image](https://user-images.githubusercontent.com/772738/161372486-7f073045-54e9-4990-8708-73e98521019a.png)